packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -111,7 +120,25 @@ const TEMPLATE_SOURCE_NPM = 'npm'
 const TEMPLATE_SOURCE_BUNDLE = 'bundle'
 const DEFAULT_TEMPLATE_VERSION = 'latest'
 
-const askApplicationExtensibiltyQuestions = (appExtensions) => {
+const LOCAL_DEV_PROJECT_DIR = 'dev'
+
+const INITIAL_QUESTION = [
+    {
+        name: 'project.type',
+        message: 'What type of PWA Kit project would you like to create?',
+        type: 'list',
+        choices: [
+            {name: 'PWA Kit App (full project)', value: 'PWAKitProject'},
+            {
+                name: 'PWA Kit Application Extension (for existing PWA Kit apps)',
+                value: 'appExtensionProject'
```

Comment:
Lets simplify this a little: 

```
What type of PWA Kit project would you like to create?
- PWA Kit Application
- PWA Kit Application Extensions
```

We don't need this "(for existing PWA Kit apps)" because I think it might server to confuse since we really want it to say something like "for _use_ with PWA-Kit Applications".

Reply:
We'll get feedback from our beta users about these questions to refine it further.